plugins/nvidia-gpu-operator/templates/wave-1/installer.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: nvidia-gpu-operator
```

Comment:
Please avoid the hardcoded name here. We should be utilizing the releaseName, please review other plugins and follow their example. This would need to be adjusted in a few diferent places like the "terra-metadata" name as well.

The make package-PLUGIN will be going off the releaseName and it must match